integration-tests/src/test/collect-coins.test.ts

```diff
@@ -61,7 +61,7 @@ describe('CollectCoins', (): void => {
                 blockchain,
                 signAccountId(deployerWallet, collector.address),
                 collector,
-                (global as any).CREDITCOIN_REUSE_EXISTING_ADDRESSES,
+                true,
```

Comment:
Why change this?

Reply:
Originally changed to `true` because the default settings were `false` and both `gate-token.test.ts` and `collect-coins.test.ts` try and register the same address. After consulting with Nathan he said it should be fine to set the default to `true`.  Reverted the change to point back to the env variable and updated the default value to `true`.